assets/src/edit-story/components/checklist/priorityChecks.js

```diff
@@ -26,12 +31,27 @@ import { StoryPosterAttached } from './checks/storyPosterAttached';
 import StoryPosterPortraitSize from './checks/storyPosterPortraitSize';
 import StoryTitleLength from './checks/storyTitleLength';
 import VideoElementMissingPoster from './checks/videoElementMissingPoster';
-import { ChecklistCategoryProvider } from './checkCountContext';
+import {
+  ChecklistCategoryProvider,
+  useCategoryCount,
+} from './checkCountContext';
+import { PanelText } from './styles';
+
+export function PriorityChecks({ isOpen, onClick, title }) {
+  const count = useCategoryCount(ISSUE_TYPES.PRIORITY);
 
-export function PriorityChecks() {
   return (
     <ChecklistCategoryProvider category={ISSUE_TYPES.PRIORITY}>
-      <div>
+      <TablistPanel
+        badgeCount={count}
+        isExpanded={isOpen}
+        onClick={onClick}
+        status={PANEL_STATES.DANGER}
+        title={title}
+      >
```

Comment:
If a category has no issues, should it be disabled? Right now a user can open a tab with no issues and see nothing :)

Reply:
I think if there's nothing in a category we shouldn't show it.

Reply:
So remove the tab completely?

Reply:
if there's nothing in the category don't show the panel, yup! Same as in the previous structure.

Reply:
https://github.com/google/web-stories-wp/pull/8089#discussion_r660691574

Just added this - the panel won't show when there are no issues for that category

Reply:
thank you so much for adding this!